src/tellor_disputables/data.py

```diff
@@ -345,7 +352,7 @@ def get_query_from_data(query_data: bytes) -> Optional[Union[AbiQuery, JsonQuery
     return None
 
 
-def get_source_from_data(query_data: bytes) -> Optional[DataSource]:
+def get_source_from_data(query_data: bytes) -> Optional[Any]:
```

Comment:
Not necessarily related to your changes in this PR, but the docstring and function name seem to be different. Docstring should say "Recreate data source" or something?